p2p/pex/pex_reactor.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/pkg/errors"
 
-	amino "github.com/tendermint/go-amino"
```

Comment:
why did this change?

Reply:
I am not sure. My main guess would be that my IDE did it in the background